CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+# Contributing
+
+This document provides guidelines for contributing to this repository.
+
+## Development Environment Setup
+
+### Prerequisites
+* Linux machine with x86-64 architecture with bash installed.
+* Basic tools: curl, tar, make.
+
+### Running the Automated Tests
+1. Clone your fork of the project.
```

Comment:
Doesn't require clone a fork so I'd probably just keep #2

Reply:
It's part of dev env setup in which you do need to clone the repo. I suppose it's not very common for someone to run the automated tests unless they want to contribute something, and for that they need a fork.

If they are not interesting in contributing then I suppose the contributing guide might not be for them.

Reply:
I would include it in another section that explains the process of contributing in a more high-level.
For example:
1. Fork this repo.
2. Make changes to the code.
3. Create a PR to the upstream o11y repo.

And then keep `Running the Automated Tests` all about running the automated tests.

Reply:
Alright. I would still add those instructions for new-comers and avoid the cloning process in the running automated instructions (as the cloning itself is not just for running the automated tests, but also for making code changes and submitting them, therefore it should be in a higher hierarchy when explaining the contribution process)

Reply:
Alright.